runtime/compiler/z/codegen/S390PrivateLinkage.cpp

```diff
@@ -3061,15 +3061,14 @@ TR::Register * J9::Z::JNILinkage::buildDirectDispatch(TR::Node * callNode)
      {
      checkException(callNode, methodMetaDataVirtualRegister, methodAddressReg);
      }
-
    OMR::Z::Linkage::generateDispatchReturnLable(callNode, codeGen, deps, javaReturnRegister, hasGlRegDeps, GlobalRegDeps);
    return javaReturnRegister;
    }
 
-////////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
 // J9::Z::PrivateLinkage::doNotKillSpecialRegsForBuildArgs -  Do not kill
 // special regs (java stack ptr, system stack ptr, and method metadata reg)
-////////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
It looks like you made an inadvertent change on this line. May I ask you to add back the two slashes that you removed from this line?